[WIP] Add content to the Observability Guide (#16)
elastic/observability-docs: docs/en/observability/get-started.asciidoc

[chapter, role="xpack"]

> [[install-observability]]

Reviewer: this file should be renamed to `install-observability`

= Get started

To use Elastic Observability, you need {es} for storing and searching your 
data, and {kib} for visualizing and managing it.

[float]
[[set-up-on-cloud]]
== Set up on Cloud

include::{docs-root}/shared/cloud/ess-getting-started.asciidoc[]

[float]
[[self-manage]]
== Self manage

Alternatively, you can install and self manage {stack-gs}/get-started-elastic-stack.html#install-elasticsearch[{es}], 
{stack-gs}/get-started-elastic-stack.html#install-kibana[Kibana], and 

> {apm-get-started-ref}/apm-server.html[APM server]. 

Author: For future improvements, I created an issue for this: https://github.com/elastic/observability-docs/issues/78

[chapter, role="xpack"]
[[install-observability]]
= Get started

> To use the Elastic Observability, you need {ess} for storing and searching your 

Reviewer: ```suggestion
To use Elastic Observability, you need {es} for storing and searching your 
```

data, and {kib} for visualizing and managing it.

[float]
[[set-up-on-cloud]]
== Set up on Cloud

To get started quickly, you can spin up a deployment of our

> https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}]. The deployment includes

Reviewer: ```suggestion
{ess-trial}[hosted {ess}]. The deployment includes
```

Author: This sentence was removed as per one of your previous comments.

> {es}, {kib}, and APM Server, and is available on AWS, GCP, and Azure.

Reviewer: I think you can get rid of this paragraph. The content is mostly repeated in the docs-root include on line 16

include::{docs-root}/shared/cloud/ess-getting-started.asciidoc[]

[float]
[[self-manage]]
== Self manage

> Alternatively, you can install and self manage {stack-gs}/get-started-elastic-stack.html#install-elasticsearch[Elasticsearch], 

Reviewer: ```suggestion
Alternatively, you can install and self manage {stack-gs}/get-started-elastic-stack.html#install-elasticsearch[{es}], 
```

{stack-gs}/get-started-elastic-stack.html#install-kibana[Kibana], and 
{apm-get-started-ref}/apm-server.html[APM server]. 
First see the https://www.elastic.co/support/matrix[Elastic Support Matrix] for information about 
supported operating systems and product compatibility. We recommend you use the same 

> version of Elasticsearch, Kibana, and APM Server.

Reviewer: ```suggestion
version of {es}, {kib}, and APM Server.
```

> 
> 
> 
> 

Reviewer: ```suggestion
```
